solr/core/src/java/org/apache/solr/handler/admin/PrepRecoveryOp.java

```diff
@@ -191,11 +195,6 @@ public void execute(CallInfo it) throws Exception {
                   }
                 }
 
-                if (coreContainer.isShutDown()) {
```

Comment:
why remove this?

Reply:
The same condition is being checked earlier in this method, - so I wasn't sure that re-checking it again is really needed. Would you suggest to restore these lines?